providers/src/airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -124,6 +124,14 @@ def add_query(self, query: str) -> None:
         """
         self.job["job"][self.job_type]["query_list"] = {"queries": [query]}
 
+    def set_queries(self, queries: list[str]) -> None:
+        """
+        Set queries for Dataproc job.
+
+        :param queries: queries for the job.
+        """
+        self.job["job"][self.job_type]["query_list"] = {"queries": queries}
```

Comment:
Instead of creating another method, I think that it would be nicer to merge it into the existing `add_query`

Reply:
Agree, updated.